cpp/src/arrow/compute/light_array_test.cc

```diff
@@ -411,6 +411,31 @@ TEST(ExecBatchBuilder, AppendBatchesSomeRows) {
   ASSERT_EQ(0, pool->bytes_allocated());
 }
 
+TEST(ExecBatchBuilder, AppendBatchDupRows) {
+  std::unique_ptr<MemoryPool> owned_pool = MemoryPool::CreateDefault();
```

Comment:
Can you a reference to the GH issue so we understand where this comes from?

Reply:
Of course, will do.

Reply:
Addressed.